src/mutable_data.rs

```diff
@@ -47,8 +55,8 @@ impl Value {
     }
 }
 
-impl std::fmt::Debug for Value {
-    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+impl Debug for Value {
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
```

Comment:
Hmm - I just noticed `std::str::from_utf8(&self.data).unwrap()` below (not part of this PR, I know).

Apart from the fact that we should always prefer the `unwrap!` macro, couldn't that cause the network to crash?  There's no requirement for the values to be valid UTF8 strings afaik, so if vaults e.g. try and log a message containing one of these, they could panic.

I think for all large binary data values we should use Andreas' [`hex_fmt` crate](https://crates.io/crates/hex_fmt), as we do [here](https://github.com/Fraser999/safe-nd/blob/master/src/lib.rs#L147) for example, or if there no real benefit in seeing the value, then just printing `..` is a good option too.